semver/can_parse.ts

```diff
@@ -2,6 +2,21 @@
 // This module is browser compatible.
 import { parse } from "./parse.ts";
 
+/**
+ * Returns true if the string can be parsed as semver.
+ *
+ * @example Usage
+ * ```ts
+ * import { canParse } from "@std/semver";
```

Comment:
I'm wondering if there is a general guideline that we can follow on whether we should put module-level import (i.e. `from "@std/semver"`) or one deeper one (i.e. `from "@std/semver/can-parse"`). Do you have any thought?

Reply:
Currently we don't have explicit policy about it

Reply:
We should use the most specific import specifier. It encourages keeping dependencies leaner and aligns with JSR.